### ISSUE TITLE
Failed to load note ... unknown color 0xef

### ISSUE THREAD

**Issue by thytruong** [philips/supernote-obsidian-plugin#29]
Hello, firstly nice work with this plugin :) Just starting using it lately.

When i try to open a supernote note with obsidian, there is the error "Failed to load note [...] unknown color 0xef"
Maybe because the note is containing some pages with the template "weekly calendar" ?

**Comment by philips** [comment]
Can you attach a note with the issue?

It is almost certainly a template issue. I have very few tests on custom templates.

**Comment by Genei180** [comment]
Hi, Great Work Indeed.

Unfortunately, i have to join the Discussion because of the same Problem.
The Note Itself just fails to Load.
The Error can only be seen on the Console, therefore it will only be Visible to an advanced user and be an annoyance too others.
This Plugin definitely doesn't deserve this.

My Error:
```
Error: unknown color 0x7f
    at RattaRLEDecoder.addColorBuffer (plugin:supernote:35452:17)
    at RattaRLEDecoder.decode (plugin:supernote:35432:18)
    at eval (plugin:supernote:35328:28)
    at Generator.next (<anonymous>)
    at eval (plugin:supernote:35272:71)
    at new Promise (<anonymous>)
    at __awaiter (plugin:supernote:35254:14)
    at eval (plugin:supernote:35323:50)
    at Array.map (<anonymous>)
    at eval (plugin:supernote:35323:35)
    at Generator.next (<anonymous>)
    at eval (plugin:supernote:35272:71)
    at new Promise (<anonymous>)
    at __awaiter (plugin:supernote:35254:14)
    at eval (plugin:supernote:35321:57)
    at Array.map (<anonymous>)
    at toImage2 (plugin:supernote:35321:32)
    at SupernoteView.onLoadFile (plugin:supernote:35731:64)
```

Might Look into it, when I have time, but can't promise.
Cant share Note, because of Personal Data Sorry, but will try too Reproduce.

**Comment by philips** [comment]
The only way someone can investigate and try and fix this issue is if someone attaches a note that reproduces it.

**Comment by Genei180** [comment]
Sorry, for not being clear.
In my understanding, the Task List Template is Natively Available on the Supernote Nomad (A6x2).

Nonetheless, I will attach an Example Note File for Debugging.
Sorry, GitHub won't let me Upload: [Personal OneDrive](https://1drv.ms/u/s!AmOpTM_ZvtIwh6JQVN-7SUiIC_W9vg?e=lDd7pc)

Might take a look at the Code if I have time tomorrow.

**Comment by Genei180** [comment]
The Error, pops up in here: [Supernote Typescript Repo, Conversion.ts Line 277](https://github.com/philips/supernote-typescript/blob/57f19b7a2a5722cadf06dbb817ab53111ad4b0e2/src/conversion.ts#L277)

For me the Color 127 pops up, which is undefined, in the Translation Parameter.
Trying too work out a solution.

**Comment by philips** [comment]
Fixed in 2.2.9
